python/grass/pygrass/modules/grid/grid.py

```diff
@@ -415,6 +422,9 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         if exc_type:
```

Comment:
If an exception occurs (input raster map doesn't exists), the cleanup temporary files function is called here.